Microsoft.Toolkit.Forms.UI.XamlHost/DpiScalingPanel.cs

```diff
@@ -4,13 +4,15 @@
 
 using System;
 using System.Diagnostics;
+using foundation = Windows.Foundation;
```

Comment:
This has nothing to do with conversion to .NET Core 3?

Reply:
It does. The same source is being compiled with the .net framework 4.6 tools and .net core, and when using .net core with support for UWP types, we have a conflict between System.Windows.* and Windows.*